src/util/misc/time.ts

```diff
@@ -0,0 +1,22 @@
+export function msToReadable(milliseconds: number, short = false): string {
+	milliseconds /= 1000 * 60; // relative time in minutes
+	const _months = Math.floor(milliseconds / (60 * 24 * 30));
```

Comment:
Why `_`?

Reply:
thats murphs thing

Reply:
@StrangeGirlMurph

Reply:
ehm good point no reason really
before I put this in its own function I had variables that were already called month/days/... I never bothered to change it back 
feel free to remove the underscore